commons/phone/phone_sms.avsc

```diff
@@ -8,6 +8,9 @@
     {"name": "timeReceived", "type": "double", "doc": "device receiver timestamp in UTC (s)"},
     {"name": "target", "type": ["null", "bytes"], "doc": "HMAC SHA-256 one-way source/target of the SMS. This hash of a given phone number will be the same unless the app is reinstalled. If the number is unknown or anonymous, this contains null.", "default": null},
     {"name": "type", "type": {"name": "PhoneSmsType", "type": "enum", "symbols": ["INCOMING", "OUTGOING", "OTHER", "UNKNOWN"]}, "doc": "direction of the SMS", "default": "UNKNOWN"},
-    {"name": "length", "type": "int", "doc": "number of characters in the message (-1 if unknown)", "default": -1}
+    {"name": "length", "type": ["null","int"], "doc": "number of characters in the message (null if unknown)", "default": null},
+    {"name": "targetIsContact", "type": ["null","boolean"], "doc": "sms sender is a known contact, null if unknown", "default": null},
+    {"name": "targetIsNonNumeric", "type": "boolean", "doc": "sms sender ID is replaced by text, e.g. a company name", "default": false},
+    {"name": "targetIsService", "type": "boolean", "doc": "target is a service, e.g. a 4-digit target", "default": false}
```

Comment:
An telco provider service (like voicemail, querying remaining MB's/minutes). We'd find it useful to differentiate between human calls/sms and automated ones. Honestly, the description of a 4-digit number probably does not work internationally. The [short code wiki page](https://en.wikipedia.org/wiki/Short_code) seems to indicate different standards between countries, and it might still not be a full list. We could simply send the number of digits here, that would be more uniformly viable.